sdk/textanalytics/ai-text-analytics/package.json

```diff
@@ -88,15 +88,15 @@
     "@azure/core-rest-pipeline": "1.0.0-beta.2",
     "@azure/core-lro": "^1.0.2",
     "@azure/core-paging": "^1.1.1",
-    "@azure/core-tracing": "1.0.0-preview.10",
+    "@azure/core-tracing": "1.0.0-preview.11",
```

Comment:
I wonder why caret is not used?

Reply:
So semi-sad thing about core-tracing is that I can't really guarantee API compatibility from one release to the next. What we're really reflecting in each core-tracing update is that we're also upgrading otel, which has _no_ API compatibility guarantees (as this entire PR demonstrates).

I think we bet (and lost) that this flux would not be long term, and thus we didn't need to build in any defenses but it's a battle we've lost at this point.

Reply:
I see, that makes sense. Hopefully we will have the desired API compatibility guarantees once otel v1.0 is released.